src/robot/romi-robot.ts

```diff
@@ -174,6 +224,14 @@ export default class WPILibWSRomiRobot extends WPILibWSRobotBase {
                 // an active WS connection, the robot is in enabled state
                 // AND we have a recent-ish DS packet
                 this._heartbeatTimer = setInterval(() => {this._setRomiHeartBeat();}, 100 );
+
+                // Set up the custom device update loop
+                setInterval(() => {
```

Comment:
For efficiency, we should probably condition the setInterval to be only if there are customDevices.